### DISCUSSION TITLE
[灌水]炫酷的绳匠ID征集

### DISCUSSION THREAD

**Discussion by muxi0330** [share121/inter-knot#777] [其他]
我好后悔啊，当初只是抱着看戏吃瓜的目的来这里，嫌麻烦就随便瞎起了个名字。现在想开始从事绳匠工作了，这名字....唉只能重新注册一个新号了，换号之前征集一波炫酷的绳匠名字!

**Comment by capacitor1** [comment]
新建绳匠

doge

**Comment by muxi0330** [comment]
这个帖子的id是777好耶

**Comment by KSN-tt41** [comment]
法厄同，绝对够拉风

**Comment by GanlinSama** [comment]
muxi0330，这个名字以后会响彻新艾利都的

**Comment by xingkong12121** [comment]
不穿内酷，绝对让人第一印象深刻，有酷，真空还拉风（doge）
